core/src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -57,6 +129,37 @@ public String getName() {
     return aliases.get(0);
   }
 
+  /**
+   * Get the display name of the {@link SettingKey}.
+   *
+   * @return The formatted display name.
+   */
+  public Component getDisplayName() {
```

Comment:
`getName`

Reply:
The original `getName` is still needed in a bunch of places so the distinction needs to be made.

Reply:
Actually I think you want a larger refactor? See my comment below.